[2025-06-15T08:44:53.466+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-06-15T08:44:53.591+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: globalsuperstore.extract.extract_and_load_to_staging scheduled__2024-11-09T00:00:00+00:00 [queued]>
[2025-06-15T08:44:53.629+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: globalsuperstore.extract.extract_and_load_to_staging scheduled__2024-11-09T00:00:00+00:00 [queued]>
[2025-06-15T08:44:53.630+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-06-15T08:44:53.921+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): extract.extract_and_load_to_staging> on 2024-11-09 00:00:00+00:00
[2025-06-15T08:44:53.941+0000] {standard_task_runner.py:63} INFO - Started process 1309 to run task
[2025-06-15T08:44:53.965+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'globalsuperstore', 'extract.extract_and_load_to_staging', 'scheduled__2024-11-09T00:00:00+00:00', '--job-id', '124', '--raw', '--subdir', 'DAGS_FOLDER/global_superstore_dw_etl.py', '--cfg-path', '/tmp/tmp7vgb7fy7']
[2025-06-15T08:44:53.976+0000] {standard_task_runner.py:91} INFO - Job 124: Subtask extract.extract_and_load_to_staging
[2025-06-15T08:44:54.069+0000] {task_command.py:426} INFO - Running <TaskInstance: globalsuperstore.extract.extract_and_load_to_staging scheduled__2024-11-09T00:00:00+00:00 [running]> on host f3cca4675eb8
[2025-06-15T08:44:55.912+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='globalsuperstore' AIRFLOW_CTX_TASK_ID='extract.extract_and_load_to_staging' AIRFLOW_CTX_EXECUTION_DATE='2024-11-09T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-09T00:00:00+00:00'
[2025-06-15T08:44:55.915+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-06-15T08:44:56.005+0000] {base.py:84} INFO - Using connection ID 'postgres' for task execution.
[2025-06-15T08:44:56.046+0000] {base.py:84} INFO - Using connection ID 'mysql' for task execution.
[2025-06-15T08:44:56.105+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.10/site-packages/***/providers/common/sql/hooks/sql.py:263 UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
[2025-06-15T08:44:56.442+0000] {logging_mixin.py:188} INFO - ✔ Đã load Category → staging.stg_Category
[2025-06-15T08:44:56.498+0000] {base.py:84} INFO - Using connection ID 'mysql' for task execution.
[2025-06-15T08:44:56.535+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.10/site-packages/***/providers/common/sql/hooks/sql.py:263 UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
[2025-06-15T08:44:59.318+0000] {logging_mixin.py:188} INFO - ✔ Đã load Customer → staging.stg_Customer
[2025-06-15T08:44:59.330+0000] {base.py:84} INFO - Using connection ID 'mysql' for task execution.
[2025-06-15T08:44:59.345+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.10/site-packages/***/providers/common/sql/hooks/sql.py:263 UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
[2025-06-15T08:45:21.078+0000] {local_task_job_runner.py:240} INFO - Task exited with return code -9
[2025-06-15T08:45:21.397+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-06-15T08:45:21.415+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
